portfolio/src/main/webapp/assets/js/voiceControl.js

```diff
@@ -0,0 +1,254 @@
+var langs =
```

Comment:
Is this a file you wrote or something provided by the lib? If it's external, I would put it in a different dir and maybe add a header with the source. It would make it easier to keep that separate and you can go back and update it as needed if you keep the link.

Reply:
I was playing around with it but never ended up using it actually - just got rid of it